_pages/documentation/general_docs/gpu_models/gpufs.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: documentation
+title: Full System AMD GPU model
+doc: gem5 documentation
+parent: gpu_models
+permalink: /documentation/general_docs/gpu_models/gpufs
+---
+
+# **Full System AMD GPU model**
+
+The Full System AMD GPU model simulates a GPU at the "gfx9" ISA level, as opposed to the intermediate language level. This page will give you a general overview of how to use this model, the software stack the model uses, and provide resources that detail the model and how it is implemented.
+
+## Requirements
+
+The Full System GPU model is primarily designed to simulate discrete GPUs using a native software stack without modification. This means that the CPU portion of simulation is not configured for detailed simulation -- Only the GPU is detailed. The [ROCm software stack](https://rocm.docs.amd.com/en/latest/) limits usage to officially supported gfx9 devices listed in the [ROCm documentation](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html).
```

Comment:
I've not tested gfx902 in GPUFS, only gfx900, gfx90a, and gfx942. I added a note about gfx900 because that is on the list as unsupported but is still working in GPUFS.

Reply:
Point taken but that list at the link also includes a bunch of GPUs we don't support in gem5, right?